app/javascript/src/views/Game.vue

```diff
@@ -656,10 +658,7 @@ export default {
             action.payload.placements.forEach((placement) => {
               provinces.add(placement.province);
             });
-          } else if (
-            action.type === 'buildFactory'
-            && this.game.buildingFactory
-          ) {
+          } else if (action.type === 'buildFactory' && this.game.buildingFactory) {
```

Comment:
Maybe we should omit this file from the diff since I think it's just linting changes/complaints from GitHub.

Reply:
Eh, this line was under the limit unless you preferred it the way it was before

Up to you, I thought it looked neater when on a single line